build.gradle

```diff
@@ -1,23 +1,28 @@
 buildscript {
   repositories {
     jcenter()
+    maven {
+      url 'https://maven.google.com/'
+      name 'Google'
+    }
+    google()
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:2.2.1'
+    classpath 'com.android.tools.build:gradle:3.1.2'
   }
 }
 
 apply plugin: 'com.android.library'
 
 android {
-  compileSdkVersion 23
-  buildToolsVersion '23.0.2'
+  compileSdkVersion 26
+  buildToolsVersion '27.0.3'
 
   defaultConfig {
     minSdkVersion 14
```

Comment:
Diego, I think it's better to update `minSdkVerston` to `16` because our `https://github.com/Wootric/WootricSDK-Android` is already using `16`. Also, new projects created below `16` won't work, so constancy will be good.